README.md

```diff
@@ -13,11 +12,13 @@ Display icons for filenames in ![Magit](https://github.com/magit/magit) buffers!
 </div>
 
 # Preview
+
```

Comment:
[MD022](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines)

Reply:
Thanks!